HKMP/HKMP.csproj

```diff
@@ -9,6 +9,7 @@
         <AssemblyVersion>0.0.0.0</AssemblyVersion>
         <TargetFramework>net472</TargetFramework>
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <LangVersion>latest</LangVersion>
```

Comment:
Could you elaborate on why you added this to the `.csproj`?

Reply:
The newest version of C# introduces some handy syntactic sugar like initializing collections or class instances with `new()` rather than e.g. `new Dictionary<ushort, GameObject>()`. I thought it might be helpful to developers.